src/dex/uniswap-v3/uniswap-v3.ts

```diff
@@ -54,6 +54,7 @@ type PoolPairsInfo = {
   fee: string;
 };
 
+const UNISWAPV3_CLEAN_NOT_EXISITING_POOL_TTL_S = 1000 * 60 * 60 * 24;
```

Comment:
extra 1000

Reply:
@mwamedacen updated value to `60 * 60 * 24`